cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -491,6 +491,20 @@ pTransferAmt =
       , Opt.help "The amount to transfer."
       ]
 
+pTreasuryWithdrawalAmt :: Parser Lovelace
+pTreasuryWithdrawalAmt =
+  Opt.option (readerFromParsecParser parseLovelace) $
+    mconcat
+      [ Opt.long "transfer"
+      , Opt.metavar "LOVELACE"
+      , Opt.help $
+          mconcat
+            [ "The amount of lovelace the proposal intends to withdraw from the Treasury. "
+            , "Multiple withdrawals can be proposed in a single governance action "
+            , "by repeating the --funds-receiving-stake and --transfer options as many times as needed."
+            ]
+      ]
```

Comment:
`pTransferAmt` and `pTreasuryWithdrawalAmt` are identical except for the help string.

So there could be a function that takes the help string as a parameter, and then have `pTransferAmt` and `pTreasuryWithdrawalAmt` call that function, for example.

It may be overkill though, up to you

Reply:
Exactly, the only reason for the new parser is the help text. But updating the existing one was not ideal because it is shared with pre-Conway MIR certificates (still present on the CLI). I think MIRs and the parser will be gone when we remove support for pre-Conway eras, so it might indeed be a little overkill to add that extra function.